### DISCUSSION TITLE
libbpf for Android

### DISCUSSION THREAD

**Discussion by ghost** [libbpf/libbpf-bootstrap#100] [Q&A]
Does this run on recent Androids?
Is there specific instructions to compile it for an Android phone?

**Comment by chenhengqi** [comment]
Please refer to [README](https://github.com/libbpf/libbpf-bootstrap#c-examples), the C examples could be built on Android with Xmake.

**Reply by kochia3** [reply]
I have the question, can we compile Rust version - libbbpf-rs projects for Android?
